pkgpanda/build/__init__.py

```diff
@@ -48,6 +48,13 @@ def run(self, cmd):
         docker.append(self.container)
         docker += cmd
         check_call(docker)
+        DockerCmd.clean()
+
+    @staticmethod
+    def clean():
+        """Removes the last container that was spawned"""
+        container_id = check_output(["docker", "ps", "--latest", "-q"]).rstrip()
```

Comment:
Another idea on how to clean this up would be to make sure all `docker run`s provide a very specific name for the container they're launching. Then we can just remove that container by it's name afterwards. What are your thoughts?